dev-utils/storybook/stories/lit-components/button.stories.js

```diff
@@ -0,0 +1,102 @@
+import { html } from "lit-element";
+import "../../../../feature-utils/poly-look/src/lit-components/buttons";
+import { themeConfiguration } from "./themeConfiguration";
+
+export default {
+    title: "Visuals/Atoms/Button",
+    component: "poly-button",
+    argTypes: {
```

Comment:
These are arguments you can use in the command line or in the story, is that correct?

Reply:
These are for the story. They tell Storybook how to render the component, but they also allow you to dynamically change them and see the result without having to go back to the code.